packages/client/src/types/config.types.ts

```diff
@@ -1,3 +1,13 @@
-import type { AxiosRequestConfig } from 'axios';
+import type { AxiosRequestConfig, AxiosRequestHeaders } from 'axios';
 
-export type Config = AxiosRequestConfig & Record<string, unknown>;
+type MockServerProps = {
+  'X-MOCK-SERVER-IGNORE': boolean;
```

Comment:
Here we need to specify that this header is optional, like this:
```suggestion
  'X-MOCK-SERVER-IGNORE'?: boolean;
```
otherwise if the developer passes in the headers object into a client, it means that it must fully comply with the headers object definition, which will contain this property as required.

Reply:
Good catch. Thank you!